internal/pkg/template/templates/workloads/services/rd-web/cf.yml

```diff
@@ -134,7 +134,7 @@ Resources:
           EgressType: VPC
           VpcConnectorArn: !Ref VpcConnector
       {{- end }}
-      {{- if and (contains .FeatureFlags "Tracing") (eq .Observability.Tracing "AWSXRAY")}}
+      {{- if eq .Observability.Tracing "AWSXRAY"}}
       ObservabilityConfiguration:
         ObservabilityEnabled: true
         ObservabilityConfigurationArn: !Sub 'arn:aws:apprunner:${AWS::Region}:${AWS::AccountId}:observabilityconfiguration/DefaultConfiguration/1/00000000000000000000000000000001'
```

Comment:
Is the ARN guaranteed to be this? can we double check

Reply:
Double checked just now - it still true that the ARN is this. This is similar to the released autoscaling default config: `arn:aws:apprunner:<region>:<account>:autoscalingconfiguration/DefaultConfiguration/1/00000000000000000000000000000001`